### PR TITLE
lib/plot: port Dropdown to PF5

### PR DESCRIPTION
**Pull request by tomasmatus**

### CODE REVIEW

File: pkg/lib/cockpit-components-plot.jsx [hunk 1]
```diff
@@ -254,22 +256,32 @@ export const ZoomControls = ({ plot_state }) => {
     if (!zoom_state)
         return null;
 
+    const dropdownItems = [
+        <DropdownItem key="now" onClick={() => { zoom_state.goto_now(); setIsOpen(false) }}>
```
Comment (cockpituous) [inline pkg/lib/cockpit-components-plot.jsx:260]: This added line is not executed by any test.

Comment (cockpituous) [inline pkg/lib/cockpit-components-plot.jsx:260]: This added line is not executed by any test.

File: pkg/lib/cockpit-components-plot.jsx [hunk 2]
```diff
@@ -254,22 +256,32 @@ export const ZoomControls = ({ plot_state }) => {
...
-                    </DropdownItem>,
-                    <DropdownSeparator key="sep" />,
-                    range_item(5 * 60, _("5 minutes")),
-                    range_item(60 * 60, _("1 hour")),
-                    range_item(6 * 60 * 60, _("6 hours")),
-                    range_item(24 * 60 * 60, _("1 day")),
-                    range_item(7 * 24 * 60 * 60, _("1 week"))
-                ]} />
+                toggle={(toggleRef) => (
+                    <MenuToggle ref={toggleRef} onClick={() => setIsOpen(!isOpen)} isExpanded={isOpen}>
```
Comment (cockpituous) [inline pkg/lib/cockpit-components-plot.jsx:276]: This added line is not executed by any test.

File: pkg/lib/cockpit-components-plot.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/lib/cockpit-components-plot.jsx:260]: This added line is not executed by any test.

File: pkg/lib/cockpit-components-plot.jsx (hunk 2 above)
Comment (cockpituous) [inline pkg/lib/cockpit-components-plot.jsx:276]: This added line is not executed by any test.

**Review by jelly — APPROVED** [review]

**Pull request merged**